README.md

```diff
@@ -1 +1,74 @@
-# Rv-041WebUI-python
\ No newline at end of file
+# RV-041WebUI/Python
+
+## Install instructions
+### Project requaire Python 2.7
+On ubuntu:
+1. Install essential for pip
+
+```sudo apt-get install python-pip python-dev build-essential```
+
+2. Install pip for python 2.7
+
+```sudo easy_install pip```
```

Comment:
You already installed `pip` on previous step (`python-pip` Ubuntu package), so it doesn't make sense to install it again.